src/App.tsx

```diff
@@ -1,61 +1,48 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { Todo } from './types/Todo';
+import { getUsersById } from './utils/getUsersById';
+
+const initialTodos: Todo[] = todosFromServer.map(todo => {
+  const preparingUser = getUsersById(todo.userId);
+
+  return {
+    ...todo,
+    user: preparingUser,
+  };
+});
```

Comment:
Ensure that the `getUsersById` function is correctly fetching user data by ID. This function should return a user object that matches the `userId` of each todo. Verify that the `getUsersById` function is implemented correctly and returns the expected user data.